### DISCUSSION TITLE
Download LKAM01

### DISCUSSION THREAD

**Discussion by sjackman** [kblin/ncbi-genome-download#187] [Q&A]
Is there an option to download assembly `LKAM01` *Picea glauca* (white spruce) mitochondrial, whole genome shotgun sequence?
https://www.ncbi.nlm.nih.gov/nuccore/?term=LKAM01
ftp://ftp.ncbi.nlm.nih.gov/sra/wgs_aux/LK/AM/LKAM01/LKAM01.1.fsa_nt.gz

```
❯❯❯ ncbi-genome-download -A LKAM01 -F fasta all
ERROR: No downloads matched your filter. Please check your options.
```

**Comment by sjackman** [comment]
Could there be an option to download all mitochondrial or plastid genomes of a given taxa?

**Comment by kblin** [comment]
ncbi-genome-download doesn't do SRA downloads, just GenBank and RefSeq. It also doesn't know about taxonomy apart from what's in the assembly summary text file. The `gimme_taxa.py` script in `contrib` can find taxids to filter for, but also doesn't know about what entries are mitochondrial/plastid.

**Comment by kblin** [comment]
That said, https://www.ncbi.nlm.nih.gov/nuccore/LKAM00000000.1 seems to be the WGS master record for it, and I'm planning to add WGS support to ncbi-acc-download soon, so that might be an option once you know the ID.

**Comment by sjackman** [comment]
> I'm planning to add WGS support to ncbi-acc-download soon, so that might be an option once you know the ID.

Excellent! I look forward to that feature.

**Comment by kblin** [comment]
I forgot about this question for a bit, sorry. Current versions of https://github.com/kblin/ncbi-acc-download/ support recursive downloads to grab WGS entries.

**Comment by sjackman** [comment]
Yeah! Thanks, Kai.

**Comment by tseemann** [comment]
@sjackman `ncbi-acc-download -v --recursive  LKAM00000000.1` seems to work.
Close this issue?
